storage/imx6ull-flash/imx6ull-flash-test.c

```diff
@@ -511,11 +511,10 @@ void test_meta(void)
 	if ((err = flashdrv_erase(dma, paddr) < 0))
 		printf("erase() failed: %d\n", err);
 
-	/* test writing page data + meta */
+	/* test writing meta + data */
 	memset(data, 0xff, pagemapsz);
-	for (i = 0; i < sizeof(aux->metadata); ++i) {
+	for (i = 0; i < sizeof(aux->metadata); ++i)
```

Comment:
not relevant change, also: against MISRA (probably not yet relevant but in the near future will be - 
Rule 15.6 - The body of an iteration-statement or a selection-statement shall be a compound-statement).

Please revert all compound-statements removal

Reply:
Thanks for pointing this out. I wasn't aware of this rule. I've restored iteration-statement braces but left out original selection-statements (`if` without braces). Should I add braces to the one-line `if` statements too?